Observability Strategy (#347)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_observability-strategy.adoc

> [id="observability-strategy_{context}"]

Reviewer: ```suggestion
[id="observability-strategy-in-service-telemetry-framework_{context}"]
```

> = Observability Strategy

Reviewer: ```suggestion
= Observability Strategy in Service Telemetry Framework
```

[role="_abstract"]

> {Project} ({ProjectShort}) does not ship any storage backends or alerting tools, but instead relies on community operators to deploy Prometheus, Alertmanager, and Elasticsearch. By default, {ProjectShort} will install subscriptions to these community operators and create instances of each application configured to work with {ProjectShort}, but you may wish to use your own deployments of these applications (or other compatible applications) instead. Setting the Observability Strategy allows you to control this behavior so that you can use alternative backends instead, in which case you do not need to configure any persistent or ephemeral storage for STF.

Reviewer: ```suggestion
{Project} ({ProjectShort}) does not include storage backends and alerting tools.  {ProjectShort} uses community operators to deploy Prometheus, Alertmanager, and Elasticsearch. By default, {ProjectShort} installs subscriptions to these community operators and creates instances of each application configured to work with {ProjectShort}. However, you can use your own deployments of these applications or other compatible applications. If you set the observability strategy so that you can use alternative backends instead, you do not need to configure persistent or ephemeral storage for {ProjectShort}.
```
